eis_toolkit/model_performance_estimation/model_performance_estimation.py

```diff
@@ -0,0 +1,35 @@
+import sklearn
+from sklearn.model_selection import KFold, LeaveOneOut, StratifiedKFold
+
+from eis_toolkit.exceptions import InvalidCrossValidationSelected, InvalidNumberOfSplit
+
+
+def performance_model_estimation(
+    cross_validation_type: str = "LOOCV", number_of_split: int = 5
```

Comment:
Literal to list all the valid cross_validation_types. Then beartype will validate also the input str